maven-resolver-provider/src/main/java/org/apache/maven/repository/internal/RelocatedArtifact.java

```diff
@@ -116,17 +116,6 @@ public Artifact setVersion(String version) {
         return new RelocatedArtifact(artifact, groupId, artifactId, classifier, extension, version, message);
     }
 
-    @Deprecated
-    @Override
-    public Artifact setFile(File file) {
```

Comment:
This will break compat, no?

Reply:
maven-resolver-provider RelocatedArtifact is present in mvn3 as well, I mean "binary compat"

Reply:
Mmh, not sure, as a default implementation is provided by `AbstractArtifact`... ?

Reply:
I've revert the removal of the two `setFile()` methods to be safe.